detox/android/detox/src/full/java/com/wix/detox/reactnative/ReactMarkersLogger.kt

```diff
@@ -7,38 +7,44 @@ import com.facebook.react.bridge.ReactMarkerConstants.*
 
 object ReactMarkersLogger : ReactMarker.MarkerListener {
 
+    private var isLoggerEnabled = false
+    
     fun attach() {
-        ReactMarker.addListener(this)
+        if (ReactNativeInfo.rnVersion().minor >= 71) {
+            isLoggerEnabled = true
+        } else {
+            ReactMarker.addListener(this)
+        }
```

Comment:
```suggestion
        if (ReactNativeInfo.rnVersion().minor >= 71) {
            ReactMarker.addListener(this)
        }
```

Reply:
@d4vidi much simpler approach! I updated the branch, thank you